packages/oid4vci-demo-frontend/src/configs/banqup-account.json

```diff
@@ -1,10 +1,11 @@
 {
   "general": {
-    "credentialName": "Bank account credential",
-    "issueCredentialType": "BankAccount",
-    "baseUrl": "http://192.168.2.90:5000/bank",
+    "credentialName": "Account credential",
+    "issueCredentialType": "Account",
+    "oid4vpAgentBaseUrl": "https://account.banqup.demo.sphereon.com/agent",
+    "oid4vciAgentBaseUrl": "https://account.banqup.demo.sphereon.com/agent",
     "verifierUrl": "https://sphereon.com",
-    "verifierUrlCaption": "Bank credential",
+    "verifierUrlCaption": "Account credential",
     "downloadUrl": "https://bank.demo.sphereon.com/agent/#/download",
     "redirectUrl": "https://knb.demo.sphereon.com"
```

Comment:
not your pr, but if the redirectUrl is old, remove it here as well

Reply:
Removed